boards/arm/efm32wg_stk3800/doc/efm32wg_stk3800.rst

```diff
@@ -0,0 +1,173 @@
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/efm32wg_stk3800/efm32wg_stk3800_defconfig``
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+The EFM32WG SoC has six gpio controllers (PORTA to PORTF), but only three are
+currently enabled (PORTB, PORTE and PORTF) for the EFM32WG-STK3800 board.
+
+In the following table, the column Name contains Pin names. E.g. PE2 means Pin number 2 on PORTE, as used in the board's datasheets and manuals.
+
++-------+-------------+-------------------------------------+
+| Name  | Function    | Usage                               |
++=======+=============+=====================================+
+| PE2   | GPIO        | LED0                                |
++-------+-------------+-------------------------------------+
+| PE3   | GPIO        | LED1                                |
++-------+-------------+-------------------------------------+
+| PB0   | GPIO        | Push Button PB0                     |
++-------+-------------+-------------------------------------+
+| PB1   | GPIO        | Push Button PB1                     |
++-------+-------------+-------------------------------------+
+| PF7   | GPIO        | Board Controller Enable             |
+|       |             | EFM_BC_EN                           |
++-------+-------------+-------------------------------------+
+| PE0   | UART0_TX    | UART Console EFM_BC_TX U0_TX #1     |
++-------+-------------+-------------------------------------+
+| PE1   | UART0_RX    | UART Console EFM_BC_RX U0_RX #1     |
++-------+-------------+-------------------------------------+
+
+System Clock
+============
+
+The EFM32WG SoC is configured to use the 48 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFM32WG SoC has three USARTs, two UARTs and two Low Energy UARTs (LEUART).
+UART0 is connected to the board controller and is used for the console.
+
+Programming and Debugging
+*************************
+
+.. note:
+   Before useing the kit the first time, you should update the J-Link firmware
+   from `J-Link-Downloads`_
+
+Flashing
+========
+
+The EFM32WG-STK3800 includes an `J-Link`_ serial and debug adaptor built into the
+board. The adaptor provides:
+
+- A USB connection to the host computer, which exposes a Mass Storage and an
+  USB Serial Port.
+- A Serial Flash device, which implements the USB flash disk file storage.
+- A physical UART connection which is relayed over interface USB Serial port.
+
+Flashing an application to EFM32-STK3800
+----------------------------------------
+
+The sample application :ref:`hello_world` is used for this example.
+Build the Zephyr kernel and application:
+
+.. code-block:: console
+
+   $ cd $ZEPHYR_BASE
```

Comment:
Alas, $ZEPHYR_BASE isn't defined until after you source zephyr-env.sh
maybe say "cd <zephyr_root_path>" as others have done.